File: 5 lang progs/prog52a.rb
puts "enter length"
length = gets.chomp
puts "enter width"
width = gets.chomp
length = length.to_f
width = width.to_f
area = length * width
perim = 2 * length + 2 * width
perim = perim.to_s
area= area.to_s
puts "the are is " + area
puts "the perim is " + perim 
/*
enter length
2
enter width
2
the are is 4.0
the perim is 8.0
*/

File: 5 lang progs/Prog52a.go
package main

import (
  "fmt"
  
)

func main() {
	fmt.Print("Enter length: ")
	var length int
	fmt.Scanln(&length)

	fmt.Print("Enter something: ")
	var width int
	fmt.Scanln(&width)
	var perimeter = 2 * (length + width)
	var area  = length * width 
    var A = fmt.Sprint(area)
	var P = fmt.Sprint(perimeter)
	fmt.Println("The area is ", A )
	fmt.Println("The width is " , P)

/*
Enter length: 2
Enter something: 2
The area is  4
The width is  8
/*
